packages/framework-integration-tests/src/sensor/health/application-child-health-indicator.ts

```diff
@@ -0,0 +1,25 @@
+import { HealthEndpoint } from '@boostercloud/framework-core'
+import {
+  BoosterConfig,
+  HealthIndicatorResult,
+  HealthIndicatorMetadata,
+  HealthStatus,
+} from '@boostercloud/framework-types'
+
+@HealthEndpoint({
+  id: 'myApplication/child',
+  name: 'My Application child',
```

Comment:
What would happen if we add this Health sensor but we don't add the parent one (`myApplication`)?

Reply:
Suppose you have the following sensors:
  A
  A/B/C

If you navigate to `/A` then only **A** sensor will be returned. But you still will be able to navigate to `/A/B/C` and get this specific sensor. If you navigate to `/A/B` it will not be found

Reply:
Aha! Ok, got it. So the "parent -> child" relationship is a loose relationship. Just like a namespace relationship. Great!